docs/mentors.md

```diff
@@ -5,6 +5,11 @@
     <th>Name</th>
     <th>Project</th>
   </tr>
+    <tr>
+    <td>Smaranjit Ghose</td>
```

Comment:
You can href your name here if you want to add a reference.

Reply:
Cool